osbs/api.py

```diff
@@ -726,28 +727,54 @@ def import_image(self, name):
 
     @osbsapi
     def get_token(self):
-        return self.os.get_oauth_token()
+        if self.os.use_kerberos:
+            return self.os.get_oauth_token()
+        else:
+            if self.os.token:
+                return self.os.token
+
+            raise OsbsValidationException("no token stored")
 
     @osbsapi
-    def login(self, token):
-        token_file = utils.get_instance_token_file_name(self.os_conf.conf_section)
-        token_file_dir = os.path.dirname(token_file)
+    def login(self, token, username, password):
+        if self.os.use_kerberos:
+             raise OsbsValidationException("can't use login when using kerberos")
+
+        if not token:
+            if username:
+                self.os.username = username
+            else:
+                self.os.username = getpass.getpass("Username: ")
+            if password:
+                self.os.password = password
+            else:
+                self.os.password = getpass.getpass()
+            self.os.use_auth = True
+            token = self.os.get_oauth_token()
+
+        if token:
```

Comment:
Why the `if-statement` then? Seems to me we should either raise an error if `token` is falsy, or not perform the check at all because something else is guaranteeing it won't be falsy.